workers/unity/Assets/Playground/Scripts/Metrics/MetricSendSystem.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+using Improbable.Gdk.Core;
+
+namespace Playground
+{
+    public class MetricSendSystem : ComponentSystem
+    {
+        private WorkerBase worker;
+
+        private float timeElapsedSinceUpdate = 0.0f;
```

Comment:
Would it be easier just to write the time of the next update and check for that?

Reply:
Doesn't make too much of a difference.

Reply:
Actually, this is better -- floating point addition is problematic if the numbers have vastly different orders of magnitude, so it gets less accurate as time goes by.

Reply:
It would be make it mildly more regular and be slightly neater I think.